Source/Core/AxisAlignedBoundingBox.js

```diff
@@ -210,6 +230,20 @@ define([
     /**
      * Determines which side of a plane this box is located.
      *
+     * @param {Plane} plane The plane to test against.
+     * @returns {Intersect} {@link Intersect.INSIDE} if the entire box is on the side of the plane
+     *                      the normal is pointing, {@link Intersect.OUTSIDE} if the entire box is
+     *                      on the opposite side, and {@link Intersect.INTERSECTING} if the box
+     *                      intersects the plane.
+     */
+    AxisAlignedBoundingBox.prototype.intersectPlane = function(plane) {
```

Comment:
On core math types, we usually only have prototype versions of functions for things like `clone` and `equals`.

Do we need this?  Is this because `intersectPlane` is being used polymorphically?  If so, this is fine.